File: 0817-design-hashmap/0817-design-hashmap.php
class MyHashMap {
    private $data;

    public function __construct() {
        $this->data = [];
    }

    public function put($key, $value) {
        $this->data[$key] = $value;
    }

    public function get($key) {
        return isset($this->data[$key]) ? $this->data[$key] : -1;
    }

    public function remove($key) {
        unset($this->data[$key]);
    }
}

File: 0817-design-hashmap/0817-design-hashmap.cs
using System.Collections.Generic;

public class MyHashMap {
    private Dictionary<int, int> data;

    public MyHashMap() {
        data = new Dictionary<int, int>();
    }

    public void Put(int key, int value) {
        data[key] = value;
    }

    public int Get(int key) {
        return data.ContainsKey(key) ? data[key] : -1;
    }

    public void Remove(int key) {
        data.Remove(key);
    }
}